docs/getting_started_on_linux.md

```diff
@@ -68,6 +68,42 @@ environment variable is set when you are building. You may find it useful to add
 `source [PATH TO VULKAN SDK]/setup-env.sh` to your `~/.bashrc` file to simplify
 environment variable setup.
 
+## Optional: Configure Git
+
+### Git SSH
+
+*   Generate SSH Key: `ssh-keygen -t rsa -b 4096 -C "EMAIL@email.com"`
+*   Add `~/.ssh.id_rsa.pub` key to GitHub
+*   Try a test connection `ssh git@github.com`
+
+### Other git config options
+
+```shell
+git config --global user.name "MY NAME"
+git config --global user.email "MY EMAIL"
+```
+
+### Clone
+
+This assumes that we are cloning into `/home/user/ireepub`. Update accordingly for
+your use.
+
+Note that if you will be cloning frequently, it can be sped up significantly by
+creating a reference repo and setting
+`IREE_CLONE_ARGS="--reference=/path/to/reference/repo"`. See
+`build_tools/scripts/populate_reference_repo.sh` for further details.
+
+```shell
+IREE_CLONE_ARGS=""
+mkdir -p /home/user/ireepub
+cd /home/user/ireepub
+git clone $IREE_CLONE_ARGS https://github.com/google/iree.git iree
+cd iree
+git submodule init
```

Comment:
Is --recursive necessary? I don't think I've ever used it

Reply:
Maybe better to use --recursive.